src/qtiCommonRenderer/renderers/interactions/PortableCustomInteraction.js

```diff
@@ -179,18 +183,20 @@ var getState = function getState(interaction) {
     return _getPciRenderer(interaction).getState(interaction);
 };
 
+var getData = function getData(customInteraction, data) {
```

Comment:
es6

Reply:
I did not change it it was only prettified